var ex = 0
try {
  throw 100
} catch (ex) {}
console.log(ex)
